completion.go

```diff
@@ -62,6 +63,11 @@ func completionsWithoutDescriptions(items []string) []Completion {
 // prefix.
 func (f *Filename) Complete(match string) []Completion {
 	ret, _ := filepath.Glob(match + "*")
+	if len(ret) == 1 {
+		if info, err := os.Stat(ret[0]); err == nil && info.IsDir() {
+			ret, _ = filepath.Glob(ret[0] + "/*")
```

Comment:
What if there is only one file that is a dir here too? I think this should just recurse? Since this has been waiting so long, I'll make the change. Thanks for contributing!!